packages/e2e/cypress/cli/cli.cy.ts

```diff
@@ -48,9 +49,134 @@ describe('CLI', () => {
+            {
+                failOnNonZeroExit: false,
+                env: {
+                    CI: true,
+                    NODE_ENV: 'development',
+                    ...databaseEnvVars,
+                },
+            },
+        )
+            .its('stderr')
+            .should('contain', 'Filtering models')
+            .should('not.contain', 'customers')
+            .should('contain', 'orders')
+            .should('not.contain', 'events')
+            .should('not.contain', 'users')
+            // .should('not.contain', 'payments')
+            .should('not.contain', 'stg_customers')
+            .should('contain', 'stg_orders')
+            .should('contain', 'stg_payments')
+            .should('contain', 'Done 🕶');
+    });
+
+    it('Should lightdash generate with --select with + postfix', () => {
+        cy.exec(
+            `${cliCommand} generate -y --project-dir ${projectDir} --profiles-dir ${profilesDir} --select stg_orders+`,
+            {
+                failOnNonZeroExit: false,
+                env: {
+                    CI: true,
+                    NODE_ENV: 'development',
+                    ...databaseEnvVars,
+                },
+            },
+        )
+            .its('stderr')
+            .should('contain', 'Filtering models')
+            .should('contain', 'customers')
+            .should('contain', 'orders')
+            .should('not.contain', 'events')
+            .should('not.contain', 'users')
+            .should('not.contain', 'payments')
+            .should('not.contain', 'stg_customers')
+            .should('contain', 'stg_orders')
+            .should('not.contain', 'stg_payments')
+            .should('contain', 'Done 🕶');
+    });
+
+    it('Should lightdash generate with --exclude', () => {
+        cy.exec(
+            `${cliCommand} generate -y --project-dir ${projectDir} --profiles-dir ${profilesDir} --exclude events`,
+            {
+                failOnNonZeroExit: false,
+                env: {
+                    CI: true,
+                    NODE_ENV: 'development',
+                    ...databaseEnvVars,
+                },
+            },
+        )
+            .its('stderr')
+            .should('contain', 'Filtering models')
+            .should('contain', 'customers')
+            .should('contain', 'orders')
+            .should('not.contain', 'events')
+            .should('contain', 'users')
+            .should('contain', 'payments')
+            .should('contain', 'stg_customers')
+            .should('contain', 'stg_orders')
+            .should('contain', 'stg_payments')
+            .should('contain', 'Done 🕶');
+    });
+
+    it('Should lightdash generate with --select and --exclude', () => {
+        cy.exec(
+            `${cliCommand} generate -y --project-dir ${projectDir} --profiles-dir ${profilesDir} --select +orders --exclude stg_orders`,
```

Comment:
can we exclude multiple models ? similar to the test you wrote with multiple `--select`

Reply:
yep, I can add that to the test